app/models/solidus_paypal_braintree/source.rb

```diff
@@ -6,6 +6,8 @@ class Source < SolidusSupport.payment_source_parent_class
     APPLE_PAY = "ApplePayCard"
     CREDIT_CARD = "CreditCard"
 
+
+    belongs_to :braintree_payment_method, polymorphic: true
```

Comment:
Still not a fan of this naming. There is no correlation between `source.payment_method` and `source.braintree_payment_method`. Which means anyone else attempting to use this needs to go through the source to figure out what the objects actually are.

There's no way we can rename `source.payment_method` without rewriting a large portion of Solidus so we should be responsible and name our association something more descriptive.

```suggestion
    belongs_to :payment_details, polymorphic: true
```
Or
```suggestion
    belongs_to :payment_source_info, polymorphic: true
```
Are much more descriptive about what the association is and what value it provides.